### DISCUSSION TITLE
How can I get the report_id in the log API?

### DISCUSSION THREAD

**Discussion by hyoung-90** [goharbor/harbor#17519] [Q&A] [answered]
Hi,

How can I get the **{report_id}** in the log API?
`GET /projects/{project_name}/repositories/{repository_name}/artifacts/{reference}/scan/{report_id}/log`

And where is the API used? Portal or any other API?

**Comment by chlins** [comment, marked as answer]
You can get the `report_id` from the response of the artifact API with `with_scan_overview` parameters.

**Comment by chlins** [comment]
If the scan status is error, user can go to see the logs that time portal will call this API.
